docs/source/frameworks/fastai.rst

```diff
@@ -0,0 +1,252 @@
+
+.. note::
+
+   The data block API is an advanced from fastai, if you prefer to use other methods
+   refers to the `main tutorial <https://docs.fast.ai/tutorial.text.html#Using-the-high-level-API>`_
+   for more information.
+
+Then, we can define a `Learner <https://docs.fast.ai/learner.html#Learner>`_ suitable for text classification in one line:
+
+.. code-block:: python
+
+   learner = text_classifier_learner(
+        imdb_dls, AWD_LSTM, drop_mult=0.5, metrics=accuracy
+    )
+
+We use the `AWD LSTM <https://arxiv.org/abs/1708.02182>`_ architecture, *drop_mult* is a parameter that controls the magnitude of all dropouts in that model, and we use `accuracy <https://docs.fast.ai/metrics.html#accuracy>`_ to track down how well we are doing. We can then fine-tune our pretrained model:
+
+.. code-block:: python
+
+   learner.fine_tune(4, 1e-2)
+
+.. code-block:: bash
+
+   epoch     train_loss  valid_loss  accuracy  time
+   0         0.453252    0.395130    0.822080  36:45
+
+We can then test out prediction:
+
+.. code-block:: python
+
+   learner.predict("I really liked that movie!")
+
+.. code-block:: bash
+
+   ('pos', TensorText(1), TensorText([0.1216, 0.8784]))
+
+
+Saving a learner with BentoML
+-----------------------------
+
+   :bdg-warning:`Warning:` ``Learner`` instance is required to save with BentoML.
+   This is a design choice to preserve functionalities provided by **fastai**.
+
+.. seealso::
+
+   Refers to :ref:`PyTorch Framework Guide<frameworks/pytorch:PyTorch>` for more information if one wants to use PyTorch model components of ``Learner`` with BentoML.
+
+   To get the PyTorch model, access it via ``learner.model``:
+
+   .. code-block:: python
+
+      import bentoml
+
+      bentoml.pytorch.save_model("my_pytorch_model", learner.model)
+
+To quickly save the trained learner, use ``save_model``:
+
+.. code-block:: python
+
+   bentoml.fastai.save_model("fastai_sentiment", learner)
+
+.. code-block:: bash
+
+   Model(tag="fastai_sentiment:5bakmghqpk4z3gxi", path="~/bentoml/models/fastai_sentiment/5bakmghqpk4z3gxi/")
+
+In addition to :ref:`general options <concepts/model:Save A Trained Model>`
+provided by :code:`save_model`, you can optionally provide a different ``pickle_module``
+for serializing the model.
+
+.. code-block:: python
+
+   bentoml.fastai.save_model("fastai_sentiment", learner, pickle_module="pickle")
+
+.. note::
+
+   By default, BentoML uses ``cloudpickle`` for serialization. The key difference from Python's ``pickle`` is that 
+   ``cloudpickle`` has the capability to serialize functions and so it can directly serialize members of the object without reference to its type.
+
+   :bdg-primary:`Our Recommendation:` ``cloudpickle`` *should be used for most cases.*
+
+
+Loading a learner with BentoML
+------------------------------
+
+To load the learner back to memory, use ``load_model``:
+
+.. code-block:: python
+
+   learner = bentoml.fastai.load_model("fastai_sentiment")
```

Comment:
Missing version in tag.

Reply:
we do accept loading without a version tag right? it will just resolve to latest